# InspectionStart.msg
string product_code
string product_name
